trunk/SUAVE/Methods/Weights/Correlations/General_Aviation/empty.py

```diff
@@ -0,0 +1,314 @@
+## @ingroup Methods-Weights-Correlations-General_Aviation
+# empty.py
+# 
+# Created:  Jan 2014, A. Wendorff
+# Modified: Feb 2016, M. Vegh
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+import SUAVE
+from SUAVE.Core import Units, Data
+from fuselage import fuselage
+from landing_gear import landing_gear
+from payload import payload
+from systems import systems
+from tail_horizontal import tail_horizontal
+from tail_vertical import tail_vertical
+from wing_main import wing_main
+from SUAVE.Methods.Weights.Correlations import Propulsion as Propulsion
+import warnings
+
+# ----------------------------------------------------------------------
+#  Empty
+# ----------------------------------------------------------------------
+## @ingroup Methods-Weights-Correlations-General_Aviation
+def empty(vehicle):
+    """ output = SUAVE.Methods.Weights.Correlations.Tube_Wing.empty(engine,wing,aircraft,fuselage,horizontal,vertical)
+        This is for a standard Tube and Wing aircraft configuration.        
+
+        Inputs:
```

Comment:
Can you inline the units?

Reply:
Also use the normal ordering of comments, like this:
        Assumptions:
        None

        Source:
        N/A

        Inputs:
        None

        Outputs:
        None

        Properties:
        None